backend/tests/postman/adminRoute.md

```diff
@@ -0,0 +1,23 @@
+## For `/admin` route
```

Comment:
this should be a json File. You need to share the entire collection. You can do so from postman